### PR TITLE
Fixes typo in readme.md and travis-tests with new EPV

### PR DESCRIPTION
**Pull request by Crizz0**

### CODE REVIEW

**Comment by Crizz0** [comment]
Die PHP Requirements sind ein wenig seltsam. Haben wir den falschen epv/ eingebunden:
`phpbb/epv dev-master requires php >=5.5.0 -> your PHP version (5.4.45) does not satisfy that requirement.`

**Comment by gn36** [comment]
The EPV problem is a result of this commit to phpbb/epv: https://github.com/phpbb/epv/commit/21f7608180175087acd2d37af100d56a41699ba5

It does not seem that the minimum version requirement is really true here, the commit simply removes older PHP Versions from the tests of the EPV (so the EPV is no longer guaranteed to work on older versions).

If we go back to a specific "release", this will result in a very old version of the epv (Oct. 2016 was 0.0.8).

Two solutions: 
1) We simply don't validate with EPV on older PHP Versions any longer, either. The EPV primarily validates the PHP Files for very basic errors and the structure of the extension. Structure does not change with PHP Version and the basic errors should also be caught with the newer versions.
2) We use a version requirement of dev-master|0.0.8, optionally also allowing the older version of the EPV. This must be done in a way that if possible dev-master is installed and only the older PHP Versions use 0.0.8.

I don't know if composer even allows combining branches and tags in constraints and how it behaves in this case, so I would actually prefer solution 1, even though this means we don't run the EPV with 5.4

**Comment by Crizz0** [comment]
Comes from the update they announced lately: https://www.phpbb.com/community/viewtopic.php?f=461&t=2430651

**Comment by Crizz0** [comment]
According to Paul we need only two need lines in the Travis.yml, see: https://www.phpbb.com/community/viewtopic.php?f=461&t=2430651&p=14810011#p14810001 

I will do this within this PR

**Comment by gn36** [comment]
If you do it like that, you will have to remove the epv dependency from composer.json, too, because our will be added by the travis script instead. Alternatively,  I guess you can change our composer.json to the result of the changed composer commands in the travis file, too and leave travis as is.

**Comment by gn36** [comment]
Our travis script is different. The PHP version running epv is hard-coded to 5.3.3, and it does not yet contain the commands to install epv. So you need to change more...

**Comment by Crizz0** [comment]
Yes, had no time for that yesterday. Will see...

File: .travis.yml [hunk 1]
```diff
@@ -51,13 +53,13 @@ install:
...
 
 before_script:
   - travis/setup-database.sh $DB $TRAVIS_PHP_VERSION
 
 script:
   - sh -c "if [ '$SNIFF' != '0' ]; then travis/ext-sniff.sh $DB $TRAVIS_PHP_VERSION $EXTNAME; fi"
   - sh -c "if [ '$IMAGE_ICC' != '0' ]; then travis/check-image-icc-profiles.sh $DB $TRAVIS_PHP_VERSION; fi"
-  - phpBB/vendor/bin/phpunit --configuration phpBB/ext/$EXTNAME/travis/phpunit-$DB-travis.xml --bootstrap ./tests/bootstrap.php
-  - sh -c "if [ '$EPV' != '0' ] && [ '$TRAVIS_PHP_VERSION' = '5.3.3' ] && [ '$DB' = 'mysqli' ]; then phpBB/ext/$EXTNAME/vendor/bin/EPV.php run --dir='phpBB/ext/$EXTNAME/'; fi"
-  - sh -c "if [ '$TRAVIS_PHP_VERSION' = '5.4' ] && [ '$DB' = 'mysqli' ]; then ../langtest/travis/test-lang.sh; fi"
```
Comment (Crizz0) [inline .travis.yml]: I think this should not be deleted, right?

File: .travis.yml [hunk 2]
```diff
@@ -51,13 +55,14 @@ install:
...
 before_script:
   - travis/setup-database.sh $DB $TRAVIS_PHP_VERSION
 
 script:
   - sh -c "if [ '$SNIFF' != '0' ]; then travis/ext-sniff.sh $DB $TRAVIS_PHP_VERSION $EXTNAME; fi"
   - sh -c "if [ '$IMAGE_ICC' != '0' ]; then travis/check-image-icc-profiles.sh $DB $TRAVIS_PHP_VERSION; fi"
-  - phpBB/vendor/bin/phpunit --configuration phpBB/ext/$EXTNAME/travis/phpunit-$DB-travis.xml --bootstrap ./tests/bootstrap.php
-  - sh -c "if [ '$EPV' != '0' ] && [ '$TRAVIS_PHP_VERSION' = '5.3.3' ] && [ '$DB' = 'mysqli' ]; then phpBB/ext/$EXTNAME/vendor/bin/EPV.php run --dir='phpBB/ext/$EXTNAME/'; fi"
+  - sh -c "if [ '$EPV' != '0' ] && [ '$NOTESTS' = '1' ]; then phpBB/vendor/bin/EPV.php run --dir='phpBB/ext/$EXTNAME/'; fi"
   - sh -c "if [ '$TRAVIS_PHP_VERSION' = '5.4' ] && [ '$DB' = 'mysqli' ]; then ../langtest/travis/test-lang.sh; fi"
```
Comment (Crizz0) [inline .travis.yml]: Seems to be only reason this is failing now.

**Comment by Crizz0** [comment]
Wieso ist der hier okay und failed dann nach Merge in master?

**Comment by gn36** [comment]
Der Lauf war schon am 24.8. vielleicht hat sich der EPV zwischendurch geändert und funktioniert so nicht mehr?

**Comment by Crizz0** [comment]
Ja, ich denke das Update war dazwischen.

**Comment by gn36** [comment]
Der Fehler liegt offenbar an einem Abhängigkeitsproblem. Vielleicht sollten wir den EPV vorerst einfach auf einen konkreten Release festnageln, z.B. 0.0.7.

**Pull request merged**